pytorch_lightning/loggers/tensorboard.py

```diff
@@ -117,15 +123,12 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace],
             from torch.utils.tensorboard.summary import hparams
             if metrics is None:
                 metrics = {}
-            exp, ssi, sei = hparams(sanitized_params, metrics)
+            exp, ssi, sei = hparams(params, {})
```

Comment:
was this not introduced in another PR recently? why pass in empty dict?

Reply:
done 5114af2